frontend/src/modules/EditZing/Components/UnmatchedGrid.tsx

```diff
@@ -43,7 +43,11 @@ export const UnmatchedGrid = ({
           <StyledUnmatchedText>
             Unmatched Students ({unmatchedStudents.length})
           </StyledUnmatchedText>
-          <MatchButton label="Match" onClick={handleMatchStudents} />
+          <MatchButton
+            label="Match"
+            onClick={matchStudents}
+            disabled={unmatchedStudents.length < 2}
+          />
```

Comment:
replace MUI button here, get rid of `MatchButton` reference entirely, and it should still have the functionality.